node/test/util/commit.js

```diff
@@ -1183,6 +1183,9 @@ a=B:Chi#a-1;Ba=a|x=U:C3-2 s=Sa:a;Bmaster=3;Os W README.md=888`,
                     status: new RepoStatus({
                         currentBranchName: "master",
                         headCommit: "2",
+                        staged: {
```

Comment:
I don't expect this change, since I don't see .gitmodules mentioned in the state.  Why is this happening?  (Indeed, I don't expect .gitmodules to ever be deleted by the actions of git-meta).

Reply:
For this test, it deletes the only submodule "s", and hence deleting .gitmodules is reasonable. But most importantly, it is not this PR  that deletes the .gitmodules, the PR only make changes to .gitmodules visible to git-meta.

Reply:
I guess I'm surprised that someone thought of that case, but maybe it's libgit2 that did so, which would be less surprising.